### DISCUSSION TITLE
We have an app that works locally, but we've just pushed it to a hosted service and the magic link is not working. When I click it I see a "login" action in the audit trail but in the app:

### DISCUSSION THREAD

**Discussion by nnitteam** [supabase/supabase#13001] [Questions]
We have an app that works locally, but we've just pushed it to a hosted service and the magic link is not working. When I click it I see a "login" action in the audit trail but in the app:

`supabaseClient.auth.user();`

returns null.

I'm not sure how to start debugging this. Any help would be appreciated.

_Originally posted by @jaredcosulich in https://github.com/supabase/supabase/discussions/7092_

**Comment by GaryAustin1** [comment]
I'm going to move this to discussions as it seems more like requesting help than an issue with the code.
Note that you are using v1 code syntax also which is not longer maintained for general bug fixes.  .user() does not exist anymore.

Also when you moved your app did you change your siteURL setting and/or adjust your redirectURL settings in the dashboard to reflect the new site address?
